django_elasticsearch_dsl/documents.py

```diff
@@ -157,14 +159,19 @@ def parallel_bulk(self, actions, **kwargs):
         return (1, [])
 
     def _prepare_action(self, object_instance, action):
-        return {
+        prepared_object = {
             '_op_type': action,
             '_index': self._index._name,
             '_id': object_instance.pk,
             '_source': (
                 self.prepare(object_instance) if action != 'delete' else None
             ),
         }
+        es_connection = get_connection()
+        if es_connection.info()['version']['number'][0] == '6':
```

Comment:
I think it is not proper as it will call to elasticsearch for knowing the version. So the indexing will be pretty slow!

Reply:
Agree. There's [another PR](https://github.com/django-es/django-elasticsearch-dsl/pull/288) addressing this issue, which includes tests, travis changes and convenient way to detect which version of Elasticsearch is used.